### PR TITLE
Shooter - Implements Networking tables

### LINKED ISSUE #86: Shooter
Tracking issue for the shooter (double flywheel so Wom in built won't be used)

### PR DESCRIPTION
**Pull request by totallysomeoneyoudontknow**
**What this PR does**

**What does it fix/add?**
Implements Networking Tables
resolves #86 

**What does this depend on?**
_References pull requests or branches/forks that this pull request depends on_

### CODE REVIEW

**Review by spacey-sooty — CHANGES_REQUESTED** [review]
delete .vs folder

File: src/main/cpp/Shooter.cpp [hunk 1]
```diff
@@ -1,77 +1,68 @@
...
+  }
 }
 
 void Shooter::SetState(ShooterState state) {
-    _state = state;
+  _state = state;
 }
 void Shooter::SetRaw(units::volt_t voltage) {
-    _rawVoltage = voltage;
+  _rawVoltage = voltage;
```
Comment (goanna247) [inline src/main/cpp/Shooter.cpp:63]: set a default

File: src/main/include/Robot.h [hunk 2]
```diff
@@ -24,10 +35,21 @@ class Robot : public frc::TimedRobot {
...
+  void SimulationInit() override;
+  void SimulationPeriodic() override;
 
  private:
- frc::EventLoop loop;
- RobotMap map;
- Shooter *shooter;
-};
\ No newline at end of file
+  behaviour::BehaviourScheduler* sched;
```
Comment (goanna247) [inline src/main/include/Robot.h:42]: remove swerve stuff

File: src/main/include/RobotMap.h [hunk 3]
```diff
@@ -1,38 +1,152 @@
...
+        shooterGearbox,
+        wom::PIDConfig<units::radians_per_second, units::volt>{"/Shooter/shooter/velocityPID/config",
+                                                               9_V / (180_deg / 1_s), 0_V / 25_deg,
+                                                               0_V / (90_deg / 1_s / 1_s)},
+        &shooterSensor,
+    };
+  };
+  Shooter shooterSystem;
+
+  struct SwerveBase {
```
Comment (goanna247) [inline src/main/include/RobotMap.h:50]: remove swerve code

File: src/main/include/Shooter.h [hunk 4]
```diff
@@ -1,43 +1,41 @@
...
+  void SetState(ShooterState state);
+  void SetRaw(units::volt_t voltage);
+  void SetPidGoal(units::radians_per_second_t);
+  std::shared_ptr<nt::NetworkTable> table = nt::NetworkTableInstance::GetDefault().GetTable("Shooter");
+  ShooterConfig GetConfig() { return _config; }
+
+ private:
+  ShooterConfig _config;
+  ShooterState _state = ShooterState::kRaw;
+  units::volt_t _rawVoltage;
```
Comment (goanna247) [inline src/main/include/Shooter.h:37]: set a deafult

**Review by goanna247 — CHANGES_REQUESTED** [review]

**Pull request merged**